server/server.go

```diff
@@ -531,6 +531,9 @@ func New(
 	log.Debugf(ctx, "Sandboxes: %v", s.ContainerServer.ListSandboxes())
 
 	s.startReloadWatcher(ctx)
+	go func() {
+		s.startReloadWatcherForMirrorRegistries(ctx, s.config.SystemContext.SystemRegistriesConfDirPath)
```

Comment:
could this behavior be configurable?

Reply:
We spoke about this with @sohankunkerkar.

We would either want this enabled by default with an option to disable this feature or disabled with an option to enable it. The latter is probably safer. Why? We have some customers using CRI-O who use or might use a lot of registry entries, and having this reload everything would be very costly.

Reply:
Yup, I like the latter. Let's disable this feature by default, and users can enable it anytime if they wish. I will need to add that option in the CRI-O config